doc/user_guide/docker/docker_usage.md

```diff
@@ -148,11 +146,6 @@ To update the password as user root run
     /root/jupyterenv/bin/jupyter-lab server password
 ```
 
-Using an internet browser you then can connect to the Jupyter server running
-in the Docker container in order to follow the tutorials presented by a set of
-Jupyter notebooks, see [Connecting to Jupyter
-Service](../user_guide.md#connecting-to-jupyter-service).
+Using an internet browser you then can connect to the Jupyter server running in the Docker container in order to follow the tutorials presented by a set of Jupyter notebooks, see [Connecting to Jupyter Service](../user_guide.md#connecting-to-jupyter-service).
 
-* `<host>`: If you daemon machine is identical to the machine your browser is running on
-then you can replace the `host` by `localhost` otherwise please use the IP address of the daemon machine.
-* `<port>`: Please use the value of environment variable `$PORT`, which refers to the port on the daemon machine you forwarded port `8888` of the Jupyter server to.
+For parameter `<host>`: If you daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
```

Comment:
```suggestion
For parameter `<host>`: If your daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
```